concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -34,13 +34,13 @@ Gauchat-1925-480-32	32	the stars	les étoiles	stēllas	12	1430	STAR
 Gauchat-1925-480-33	33	One does not have	Il ne faut pas	fallit	12		
 Gauchat-1925-480-34	34	to have	avoir	habēre	12	773	HAVE
 Gauchat-1925-480-35	35	fear	peur	pavōre	12	781	FEAR (FRIGHT)
-Gauchat-1925-480-36	36	from the thunder	du tonnerre	tonĭtru	12		
+Gauchat-1925-480-36	36	(of) the thunder	du tonnerre	tonĭtru	12	1150	THUNDER
 Gauchat-1925-480-37	37	In the month	Au mois	mēse	14		
 Gauchat-1925-480-38	38	of February	de février	febrariu	14		
 Gauchat-1925-480-39	39	there is	il y a	habet	14	2152	THERE IS OR THERE ARE
 Gauchat-1925-480-40	40	snow	de la neige	nĭve	14	784	SNOW
 Gauchat-1925-480-41	41	ice	de la glace	glacia	14	617	ICE
-Gauchat-1925-480-42	42	It is (2)	Il fait	facit	14		
+Gauchat-1925-480-42	42	It is	Il fait	facit	14		
```

Comment:
Important! all instances of (2) NEED to stay! We need to disambiguate glosses which occur more than one time.